include/outputs

```diff
@@ -169,14 +169,17 @@ textFail(){
   colorcode="$BAD"
   while read -r i; do
```

Comment:
What is this `i`?

I think a more expressive name would be better.

Reply:
changed, variable is called excluded_item instead